.travis.yml

```diff
@@ -0,0 +1,34 @@
+notifications:
+  email:
+    on_success: never
+language: cpp
+sudo: false
+cache:
+  ccache: true
+  pip: true
+  directories:
+  - /usr/local
+before_cache:
+  - brew cleanup
+os:
+  - osx
+env:
+  global:
+    - NINJA_STATUS="[%p %u/%t@%o %r]"
+    - PROJECT_NAME=${PWD##*/}
+    - CMAKE_PREFIX_PATH=/usr/local/opt/qt5:$CMAKE_PREFIX_PATH
+  matrix:
+  - BUILD_TYPE=Debug
+  - BUILD_TYPE=Release
+before_install:
+  - brew update
+  - brew outdated cmake || brew upgrade cmake
+  - brew install cppcheck doxygen ninja
+  - brew install zeromq python qt5 homebrew/science/hdf5
+  - pip install pyparsing
+  - ssh-keyscan bbpcode.epfl.ch >> ~/.ssh/known_hosts
```

Comment:
It doesn't. It waits for the user to say y/n to add the key to the known_hosts indefinitely.